modules/delegates/helpers/getDelegatesCounts.ts

```diff
@@ -0,0 +1,56 @@
+import { SupportedNetworks } from 'modules/web3/constants/networks';
+import { AllDelegatesEntry, DelegateRepoInformation } from '../types';
+import { add } from 'date-fns';
+import { getNewOwnerFromPrevious } from 'modules/migration/delegateAddressLinks';
+
+export default function getDelegatesCounts(
+  githubDelegates: DelegateRepoInformation[] | undefined,
+  allDelegateAddresses: AllDelegatesEntry[],
```

Comment:
Same, joining delegates into one single entity would make things easier. You can also get the counts by filtering by the property "recognized"